api/turing/web/web.go

```diff
@@ -7,6 +7,9 @@ import (
 
 func FileHandler(path string) http.Handler {
 	fn := func(w http.ResponseWriter, r *http.Request) {
+		// Ref: https://create-react-app.dev/docs/production-build/#static-file-caching
+		w.Header().Set("Cache-Control", "no-cache, no-store, must-revalidate")
+		w.Header().Set("Expires", "0") // For proxies
```

Comment:
This `FileHandler` function is generic one and is not specific to React or CRA. For example, it's also being used for serving `swagger.yaml` file. 

It would be better to parametrize it  with either `disableCahche bool` or `headers http.Header` arguments.